__tests__/app.test.js

```diff
@@ -445,3 +445,41 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("should respond with an array of users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { users } = body;
+        expect(Array.isArray(users)).toBe(true);
+      });
+  });
+
+  test("each user should have the required properties", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { users } = body;
+        expect(users).toHaveLength(4);
+        users.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
+  });
+
+  test("should respond with a 404 status code with invalid endpoint", () => {
```

Comment:
This endpoint is an invalid end point so would not fall into the controller of the endpoint /api/users.
Therefore a test for an invalid end point such as this, would be in a separate describe block. 

Something like /api/articles/monkey however would fall into the /api/articles/article_id end point, and would therefore be dealt with in the /api/articles/article_id test block. Hope that makes sense !

Reply:
Nice work Becky!

Reply:
Thanks Hannah! That does make sense :)